.github/workflows/cd-cli.yml

```diff
@@ -32,83 +31,38 @@ jobs:
-
-      - name: Publish to npm
-        run: |
-          git diff
-          pnpm --filter ${{ env.PACKAGE_NAME }} exec \
-            oclif manifest
-          pnpm --filter ${{ env.PACKAGE_NAME }} publish \
-            --access public --provenance --no-git-checks --tag ${{ steps.deployment.outputs.release-channel }}
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Compile the CLI into a single executable
+        run: pnpm -r compile
 
-      - name: Build package code and produce tarballs
-        run: |
-          pnpm --filter ${{ env.PACKAGE_NAME }} run \
-            build:tar
-
-      # https://github.com/oclif/oclif
-      # https://github.com/oclif/oclif/blob/61cb7664ce38f1b7827ff652f063c3be7adeb1af/src/tarballs/config.ts
-      - name: Publish tarballs to storage
-        run: |
-          cd ${{ env.PACKAGE_PATH }}
-          pnpm oclif upload tarballs \
-            --root tmp
-          if [ "${{ steps.deployment.outputs.release-channel }}" = "latest" ]; then channel="stable"; else channel="${{ steps.deployment.outputs.release-channel }}"; fi
-          pnpm oclif promote \
-            --root tmp \
-            --channel $channel \
-            --version ${{ steps.deployment.outputs.release-version }} \
-            --sha ${{ steps.deployment.outputs.release-sha }} \
-            --indexes
-
-      - name: Copy installation scripts to storage
-        run: |
-          cd ${{ env.PACKAGE_PATH }}
-          aws s3 cp ./scripts s3://${{ env.S3_BUCKET }} --recursive
-
-      - name: Invalidate storage CDN
-        run: |
-          aws cloudfront create-invalidation --distribution-id ${{ env.CLOUDFRONT_DISTRIBUTION }} --paths "/*"
+      - uses: actions/upload-artifact@master
+        with:
+          name: ${{ env.ARTIFACT_NAME }}-${{ matrix.os }}
+          path: ${{ env.PACKAGE_PATH }}/dist
+  end-release:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - uses: actions/download-artifact@master
+        with:
+          name: ${{ env.ARTIFACT_NAME }}-ubuntu-latest
+          path: ${{ env.PACKAGE_PATH }}/dist
+      - uses: actions/download-artifact@master
+        with:
+          name: ${{ env.ARTIFACT_NAME }}-macos-latest
+          path: ${{ env.PACKAGE_PATH }}/dist
+      - uses: actions/download-artifact@master
+        with:
+          name: ${{ env.ARTIFACT_NAME }}-windows-latest
+          path: ${{ env.PACKAGE_PATH }}/dist
 
       - name: Upload artifacts to release
         uses: softprops/action-gh-release@v2
-        if: ${{startsWith(github.ref, 'refs/tags/') }}
         with:
+          make_latest: true
+          prerelease: true
+          name: cli
+          tag_name: cli-${{ steps.deployment.outputs.release-version }}
           files: |
-            ${{ env.PACKAGE_PATH }}/tmp/dist/*
-
-      - uses: docker/setup-qemu-action@v3
-
-      - uses: docker/setup-buildx-action@v3
```

Comment:
Why delete the docker deployments? 
Maybe add a comment/issue to re do it.. necessary for some high order wrappers like the circle ci orb

Reply:
i started from scratch because i got a bit confused by the amount of jobs.